CMSgov/design-system
# [WNMGDS-921] Add link variables for theming

## Summary
Adding variables to use to style links to make the design system more theme-able by child design system themes.

### Added
- Added component variables for styling default and inverse links

### Changed
- Changed previous component styles to use new component variables. For example, `color: $color-primary;` becomes `color: $link-color;`

## How to test
- Go to http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-921-add-link-theming and compare links to the live design.cms.gov website
- Test in child design system themes (there should be no visual impact)
